options/path.c

```diff
@@ -76,6 +76,11 @@ static const char *mp_get_platform_path(void *talloc_ctx,
     assert(talloc_ctx);
 
     if (global->configdir) {
+        // Return early if configdir is an empty string, which can only happen
+        // if opts->load_config == false.
```

Comment:
this doesn't really capture what effect this has, namely: when config is disabled all platform paths become NULL, not just the ones in `config_dirs`

Reply:
Changed